src/azul/__init__.py

```diff
@@ -418,6 +418,18 @@ def access_token_audience_list(self):
             f"{self.access_token_issuer}/userinfo"
         ]
 
+    @property
+    def hmac_key(self):
```

Comment:
Properties that return a constant value don't need to be properties. A class attribute as in 

```
hmac_key = "…"
```

suffices.